src/stare/models/plot.py

```
@@ -0,0 +1,139 @@
+    reference_code: str | None = None
+    status: LenientPlotStatus | None = None
+    short_title: str | None = None
+    full_title: str | None = None
+    groups: Groups | None = None
+    documentation: Documentation | None = None
+    analysis_team: Team = Field(default_factory=Team)
+    metadata: Metadata | None = None
+    superseded_by: list[RelatedPublication] = Field(default_factory=list)
+    phase1: PlotPhase1 | None = None
+
+    def __rich__(self) -> Panel:
```

Ruff (0.16.0)

[warning] Too many branches (16 > 12)

(PLR0912)